### PR TITLE
Fix airplane mode warning and other warnings

### LINKED ISSUE #957: Airplane mode warning seen in logcat
Not a major issue, but when I use Picasso to load an image, I'm seeing this warning in logcat:
```
W/Settings( 9157): Setting airplane_mode_on has moved from android.provider.Settings.System to android.provider.Settings.Global, returning read-only value.
```

I'm using Picasso 2.5.2 (the latest release as of this writing).

### PR DESCRIPTION
**Pull request by dnkoutso**
closes #957

### CODE REVIEW

File: picasso/src/main/java/com/squareup/picasso/Utils.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
 import static android.os.Build.VERSION_CODES.HONEYCOMB;
 import static android.os.Build.VERSION_CODES.HONEYCOMB_MR1;
 import static android.os.Process.THREAD_PRIORITY_BACKGROUND;
-import static android.provider.Settings.System.AIRPLANE_MODE_ON;
+import static android.provider.Settings.Global.AIRPLANE_MODE_ON;
```
Comment (JakeWharton) [inline picasso/src/main/java/com/squareup/picasso/Utils.java]: We still need to use `System.AIRPLANE_MODE_ON` on pre-API 17 though 😢

Comment (dnkoutso) [inline picasso/src/main/java/com/squareup/picasso/Utils.java]: :(

Comment (dnkoutso) [inline picasso/src/main/java/com/squareup/picasso/Utils.java]: Will take care of it.

**Comment by dnkoutso** [comment]
Updated, cleaned up some other deprecations...

**Comment by JakeWharton** [comment]
Awesome! LGTM

**Comment by dnkoutso** [comment]
Live by: screen_shot_2016-06-09_at_10_37_55_pm

**Comment by JakeWharton** [comment]
haha :+1: :+1:

**Pull request merged**